api/test/integration/test_rbac_white_agent_endpoints.tavern.yaml

```diff
@@ -1859,3 +1859,21 @@ stages:
         Authorization: "Bearer {test_login_token}"
     response:
       <<: *permission_denied
+
+---
+test_name: GET /agents/uninstall
+
+stages:
+
+  - name: Check user's permission to uninstall agents (Allowed)
+    request:
+      verify: False
+      url: "{protocol:s}://{host:s}:{port:d}/agents/uninstall"
+      method: GET
+      headers:
+        Authorization: "Bearer {test_login_token}"
+    response:
+      status_code: 200
+      json:
+        message: "User has permission to be uninstalled"
```

Comment:
```suggestion
        message: "User has permission to uninstall agents"
```

Reply:
Changed in b7dcc227e48732b77db2c92c1087e322a55d8ae9.